### ISSUE TITLE
NTR: Potassium high in blood when symptomatic (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8391]
New term request Potassium high in blood when symptomatic (elthat_151021164314):

**Comment by pnrobinson** [comment]
@MickeySegal
This possibly related term exists already -- [Triggered by ingestion of potassium-rich food HP:0031167](https://hpo.jax.org/app/browse/term/HP:0031167)
I wonder if the new term should be "Triggered by hyperekalemia" with an analogous definition or saying that typically blood chemistry taken during an attack will reveal hyperkalemia?
What disease is being annotated?

**Comment by MickeySegal** [comment]
We use this finding for conditions such as https://www.ncbi.nlm.nih.gov/books/NBK459165/, which digitalis "inhibits the ATPase resulting in increased intracellular sodium levels. The build-up of intracellular sodium leads to a shift of sodium extracellularly through another channel in exchange for calcium ions" and "inhibition of the sodium-potassium ATPase leads to hyperkalemia and can be used as a marker of toxicity severity".  So it is not so much a trigger as an additional effect. 

For HP:0031167 we've kept the name as "Potassium supplements worsen attacks" because potassium rich foods such as potatoes don't trigger attacks because they also contain carbohydrate, which lowers serum potassium.  The only reliable way of increasing potassium orally is with supplements or other medications.  

So I think we need a separate finding about blood levels, and for all diseases with the finding "Potassium supplements worsen attacks" we also include the finding "Potassium high in blood when symptomatic".  The concepts are not identical - "Potassium supplements worsen attacks" is a subset of "Potassium high in blood when symptomatic".

**Comment by pnrobinson** [comment]
It seems to me that "Potassium high in blood when symptomatic" is not necessarily true with digitalis toxicity, in fact, toxicity can be triggered by hypokalemia. So it seems there is a more complicated relationship between K and symptoms. I am not an expert here, but a quick search did not seem to find any systematic relationship, and I did not find any other diseases with such a relationship. I would suggest we not create a term for this because there is a good chance for confusing people and also of suggesting a possibly false statement that digitalis toxicity is only symptomatic if K is high.

**Comment by MickeySegal** [comment]
A better example is hyperkalemic periodic paralysis: https://www.ncbi.nlm.nih.gov/books/NBK1496/
"an increase of serum potassium concentration of at least 1.5 mmol/L during an attack of weakness"

**Comment by pnrobinson** [comment]
reopening

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000833
- label: Hyperkalemia while symptomatic
- parents: [Abnormal blood potassium concentration (HP:0011042)](https://hpo.jax.org/app/browse/term/HP:0011042)
- pmid: PMID:20301669
- def: Increased concentration of potassium in the blood circulation specifically measured during a disease flareup, attack, or exacerbation.
- comment: For instance, in hyperkalemic periodic paralysis, one may observe an increase of serum potassium concentration of at least 1.5 mmol/L during an attack of weakness.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8391
- synonyms: Hyperkalemia during attack(NONE)
